backend/events/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from . import views
+
+# URL conf
+urlpatterns = [
+    path("", views.EventAPIView.as_view(), name="get_events"),
```

Comment:
```suggestion
    path("", views.EventAPIView.as_view(), name="list_events"),
    path("<uuid:pk>/", views.EventAPIView.as_view(), name="get_event"),
```

Reply:
@DevilsAutumn are you sure, we should be exposing our primary keys via link ??

Reply:
we should try I am some sort of serialization over the existing UUID if we are putting them over the URLS?
`hashing` maybe?

Reply:
how are we going to retrieve one single event then? 🤔 I dont think there is any problem.

Reply:
> we should try I am some sort of serialization over the existing UUID if we are putting them over the URLS? `hashing` maybe?

or maybe simply overriding get_permissions to allow only authenticated user to PUT, PATCH, DELETE?

Reply:
Seems good !